lib/src/extensions.dart

```diff
@@ -110,6 +106,18 @@ extension DateTimeExtensions on DateTime {
         totalMinutes ~/ 60,
         totalMinutes % 60,
       );
+
+  /// Returns [DateTime] without timestamp.
+  DateTime get withoutTime => DateTime(year, month, day);
+
+  /// Compares time of two [DateTime] objects.
+  bool hasSameTimeAs(DateTime other) {
```

Comment:
there one function already available for the same purpose `isAtSameMomentAs`, any other reason for adding this ?

Reply:
We need to check only time in DateTime object regardless of what date is there. `isAtSameMomentAs` does not provide that.